### DISCUSSION TITLE
OnlyT 2.2.0.1

### DISCUSSION THREAD

**Discussion by AntonyCorbett** [AntonyCorbett/OnlyT#487] [Announcements]
Support for NDI® output. Please treat this pre-release as experimental!

To enable NDI support, please run OnlyT with **--ndi** on the command-line. When enabled, the Timer window is not visible; rather the output is available as an NDI source. If this release is successful, I will also make the countdown timer optionally available as a separate NDI source.

Note that NDI® is a registered trademark of Vizrt Group, Inc.

Note that this release is 64-bit only (whereas the previous release was 32-bit). This release will appear as a different installation of OnlyT but will share settings with any 32-bit installation you may have.

<hr /><em>This discussion was created from the release <a href='https://github.com/AntonyCorbett/OnlyT/releases/tag/2.2.0.1'>OnlyT 2.2.0.1</a>.</em>

**Comment by KovalevArtem** [comment]
Please add NDI support for Meeting Countdown 🙏
